h2o-py/h2o/estimators/infogram.py

```diff
@@ -792,6 +843,23 @@ def relevance_index_threshold(self):
         which gets set to 0.1.
 
         Type: ``float``, defaults to ``-1.0``.
```

Comment:
Same here. First line says number between 0-1,... Default -1 sets to 0.1..

Reply:
I think @valenad1 has a good point.  We could just set those values to 0.1 as default instead of setting it to -1 and then set it to 0.1.

Reply:
Let me make a quick change in a new PR.